glommio/src/executor/stall.rs

```diff
@@ -0,0 +1,305 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the MIT/Apache-2.0 License, at your convenience
+//
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2021 Datadog, Inc.
+//
+#![allow(dead_code)]
```

Comment:
this is not ideal. If you absolutely must mask something, best to mask it at the use-site level, not the file level